mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -157,7 +166,15 @@ def playlistclear(context, name):
 
         Clears the playlist ``NAME.m3u``.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
```

Comment:
> Though perhaps we could just add a mpd/fallback_playlist_provider setting to at least make it explicit?

That's may be best idea, that way I can let it default to m3u backend in config and call any error user's fault :) Is m3u backend enabled by default?